src/main/java/gift/repository/ProductDao.java

```diff
@@ -25,37 +25,39 @@ private static RowMapper<Product> getProductRowMapper() {
             String name = rs.getString("name");
             Long price = rs.getLong("price");
             String imageUrl = rs.getString("imageUrl");
-            return new Product(id, name, price, imageUrl);
+            Boolean acceptedByMD = rs.getBoolean("acceptedByMD");
+            return new Product(id, name, price, imageUrl, acceptedByMD);
         };
     }
 
```

Comment:
```suggestion
private final RowMapper<Product> productRowMapper = (rs, rowNum) -> {
        Long id = rs.getLong("id");
        String name = rs.getString("name");
        Long price = rs.getLong("price");
        String imageUrl = rs.getString("imageUrl");
        Boolean acceptedByMD = rs.getBoolean("acceptedByMD");
        
        return new Product(id, name, price, imageUrl, acceptedByMD);
    };
```

getProductRowMapper() 메서드 선언 부분에 코멘트가 안 남겨져서 여기에 남겨요. 

이 메서드는 static일 이유가 없어 보이네요 ㅎㅎ 그리고 함수가 아닌 변수로 선언하는 게 더 편할 것 같아요!
중복된 mapping 로직을 분리한 것은 좋습니다 👍